app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java

```diff
@@ -1458,6 +1458,27 @@ public void onEvent() {
         });
     }
 
+    public long getDuration() {
```

Comment:
There is a few usages of mVideoManager.getDuration in this same file (seek functions). Should those be changed to use this one?

Reply:
I replaced mVideoManager.getDuration where appropriate, and changed getDuration and getBufferedPosition to fallback to 0 instead of -1. I also moved the logic for calculating a safe skip/seek position into a utility so it can be used by `skip()` and `seek()`